examples/truffle/contracts/OverloadedMethods.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @dev Contract to illustract how to work around overloaded Solidity methods.
```

Comment:
"illustract" is this a pun or a typo?

Reply:
I want to say its a pun, but its actually a typo.